build/configs/rtl8720e/hello/defconfig

```diff
@@ -127,8 +127,8 @@ CONFIG_ARCH_CHIP_RTL8720E=y
 CONFIG_RTL8720E_UART=y
 CONFIG_RTL8720E_UART0=y
 CONFIG_RTL8720E_UART1=y
-CONFIG_RTL8720E_UART2=y
-CONFIG_RTL8720E_UART3=y
+# CONFIG_RTL8720E_UART2 is not set
+# CONFIG_RTL8720E_UART3 is not set
```

Comment:
This is still modified manually. When I turn them off using menuconfig, other configs are modified together due to dependencies. Please update this using menuconfig and then verify them with changed.

Reply:
I have pushed, and fixed some defconfig files, the hello config now matches the .config of the generated menuconfig